packages/components-js/src/examples/checkbox-example.html

```diff
@@ -0,0 +1,16 @@
+<form>
+  <p-checkbox name="some-name" label="Some Label" value="some-label"></p-checkbox>
+  <button type="submit">Submit</button>
+</form>
+
+<p-text>Last submitted data: </p-text>
+
+<script>
+  const debugElement = document.querySelector('p-text');
+  const form = document.querySelector('form');
+  form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    const formData = new FormData(form);
+    debugElement.innerText = `Last submitted data: ${formData.get('some-name')}`;
```

Comment:
This will currently show `Last submitted data: null` when the checkbox is unchecked, maybe better to just show nothing (`Last submitted data: `) instead of `null`.